v4-cokapi/cokapi.js

```diff
@@ -37,7 +37,7 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 var IS_DEBUG = false;
 
-var PRODUCTION_PORT = 3000;
+var PRODUCTION_PORT = 3001;
```

Comment:
Because if the runner of this service changes the port then he would need to change the fronted URL and port too.